src/schema/artwork/meta.ts

```diff
@@ -32,32 +34,45 @@ const ArtworkMetaType = new GraphQLObjectType({
           defaultValue: 155,
         },
       },
-      resolve: (artwork, { limit }) =>
-        truncate(
-          join(", ", [
-            partnerDescription(artwork),
-            artistNames(artwork),
-            titleWithDate(artwork),
-            artwork.medium,
-            dimensions(artwork),
-          ]),
-          limit
-        ),
+      resolve: (artwork, { limit }) => {
+        const fields = [
+          partnerDescription(artwork),
+          artistNames(artwork),
+          titleWithDate(artwork),
+          artwork.medium,
+          dimensions(artwork),
+        ]
+
+        const description = truncate(join(", ", fields), limit)
+        return description
+      },
     },
     image: {
       type: GraphQLString,
-      resolve: ({ images }) =>
-        setVersion(getDefault(images), ["large", "medium", "tall"]),
+      resolve: ({ images }) => {
+        return setVersion(getDefault(images), ["large", "medium", "tall"])
+      },
+    },
+    share: {
+      type: GraphQLString,
+      resolve: artwork => {
+        return join(", ", [
```

Comment:
Why not use `Array.prototype.join()`?

Reply:
This uses our [helper](https://github.com/artsy/metaphysics/blob/876d7346a9da828397850094b0fee10f85e5a3fc/src/lib/helpers.ts#L57) which does, but also removes null's (not sure if totally necessary, but /shrug)

Reply:
I see. The name could be improved a bit then, imo.